lib/wallaby/metadata.ex

```diff
@@ -1,9 +1,5 @@
 defmodule Wallaby.Metadata do
-  @moduledoc """
-  Metadata is used to encode information about the browser and test. This
-  information is then stored in a User Agent string. The information from the
-  test can then be extracted in the application.
-  """
```

Comment:
I felt bad deleting that documentation, so I think a code comment is a great idea. 👍